examples/platform/esp32/common/TimeSync.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <esp_err.h>
+#include <esp_log.h>
+
+#if CONFIG_ENABLE_SNTP_TIME_SYNC
+#include <esp_sntp.h>
+
+static const char * TAG = "ESP_TIME_SYNC";
+
+#define REF_TIME 1546300800 /* 01-Jan-2019 00:00:00 */
+/* Timer interval once every day (24 Hours) */
+#define TIME_PERIOD (CONFIG_SNTP_SYNC_INTERVAL_DAY * 86400000ULL)
```

Comment:
add interval to names of things. Is this _SEC or something else?

Reply:
Done